### DISCUSSION TITLE
Background color doesn't change!!

### DISCUSSION THREAD

**Discussion by AnassAb** [tailwindlabs/tailwindcss#9134] [Help] [answered]
Hello everyone, 
I'm trying to build a simple application using tailwindcss instead of vanilla css but the bg-[color] class doesn't work for me I'm not sure where is the problem, I tried using it with custom colors & original colors from the docs (also when use devtools to inspect the classes don't show up) but it works just fine with vanilla css.
-tailwind.config.js:
```
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
    fontFamily: {
      Nunito: ["Nunito Sans", "sans-serif", "monospace"],
    },
    screens: {
      sm: "480px",
      md: "768px",
      lg: "976px",
      xl: "1440px",
    },
    colors: {
      "blue-dark": "hsl(209, 23%, 22%)",
      "blue-dark-bg": "hsl(207, 26%, 17%)",
      "blue-dark-light": "hsl(200, 15%, 8%)",
      "gray-dark": "hsl(0, 0%, 52%)",
      "gray-light-bg": "hsl(0, 0%, 98%)",
    },
  },
  plugins: [],
};
```
- HTML code:
```
<main class="px-14 py-6 bg-gray-light-bg">
  <div class="bg-white">
      hello
  </div>
</main>
```

**Comment by wongjn** [comment, marked as answer]
If the classes don't show up in CSS in devtools, then it sounds like your HTML code is not covered by the `content` globs, or there has not been a CSS compilation run after the HTML code file was updated.

I can see from this exact example, `bg-white` will not generate a class here because the `white` color no longer exists since you have completely overridden `colors` by placing the key as `theme.colors`. If you wanted to **extend** the default colors that come with Tailwind instead of **replacing** them, you should place your color configuration in `theme.extend.colors`. If you did this, `bg-white` would work here.

**Reply by AnassAb** [reply]
Thanks for the answer, it was my config file that was causing the problem, I should've put my custom colors in `theme.extend` now it works fine, thanks again!
